### DISCUSSION TITLE
账号无法登录...

### DISCUSSION THREAD

**Discussion by wuwuyi551** [lanyeeee/picacomic-downloader#22] [Q&A] [answered]
这是没挂梯子的原因吗<img width="265" alt="捕获" src="https://github.com/user-attachments/assets/c3fec0fe-844f-43bd-a290-c2c1b9fb2db2">

**Comment by lanyeeee** [comment, marked as answer]
是

**Reply by wuwuyi551** [reply]
挂梯子后能用了，不过很容易闪退
